library/linkedList.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+class Node {
+  constructor() {
+    this.data = 1;
+    this.next = null;
+  }
+
+  addLink() {
+    let nextNode = this.next;
+    if (nextNode === null) {
```

Comment:
you can remove this if, the while is enough

Reply:
removed